private/buf/bufconvert/bufconvert.go

```diff
@@ -30,10 +30,16 @@ import (
 const (
 	// MessageEncodingBin is the binary image encoding.
 	MessageEncodingBin MessageEncoding = iota + 1
+	// MessageEncodingText is the text image encoding.
+	MessageEncodingText
 	// MessageEncodingJSON is the JSON image encoding.
 	MessageEncodingJSON
 	// formatBin is the binary format.
 	formatBin = "bin"
+	// formatBinProto is also used for binary proto format.
+	formatBinProto = "binproto"
```

Comment:
I think this might raise some questions around being able to do things more than one way seeing that this is the same as just bin

Reply:
Yes - this is effectively just an alias for `bin`, which isn't necessary - we only have one way to do things here, and we chose the name `bin`. This should be deleted.